### DISCUSSION TITLE
PCE Turbo Buttons

### DISCUSSION THREAD

**Discussion by haydnkilyan** [darthcloud/BlueRetro#663] [General]
I love how you can set the speed of the turbo button presses, does anyone know which setting mimics the original PCE turbo switches the closest? I have it set on 1/2 frames at the moment, not sure it feels right?

**Comment by darthcloud** [comment]
On PC-Engine/TG16 original controller the slow turbo is the equivalent of "4/8 frames" and the fast one is "2/4 frames".

**Comment by haydnkilyan** [comment]
Wonderful! Thank you
